### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by angrycaptain19**
Updates readme to reflect new code review functionality

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,50 +1,31 @@
...
-        args: [--diff=git diff HEAD, --no-summary]
-```
+## IDE Coding Assistant
 
-Note: Sourcery is fully free for open-source projects - but you need a [Team plan](https://docs.sourcery.ai/Product/Plans/) to use Sourcery in CI or as a pre-commit hook for private projects.
-
-## IDE Integration
-
-Sourcery integrates into your editor to review and improve your code while you’re writing it.
+In addition to code reviews, you can get
```
Comment (brendanator) [inline README.md:28]: **🔨 issue:** This section isn't finished‏

Comment (Hellebore) [inline README.md:28]: this sentence needs finishing

File: README.md [hunk 2]
```diff
@@ -55,25 +36,18 @@ Find quick installation instructions here:
...
-Or to try it out right away, ⭐ this repo, and Sourcery will review your most popular Python repo and send you a pull request. As an added bonus **@sourcery-ai-bot** will ⭐ your repo too.
-
-## Demo
-
-![Sourcery demo](sourcery-demo.gif)
+We're always trying to learn how we can make Sourcery better. Let us know any thoughts you have by emailing tim@sourcery.ai
 
 ## **Links**
 
-- [Docs](https://docs.sourcery.ai/)
```
Comment (Hellebore) [inline README.md:72]: why remove the docs?

File: README.md [hunk 3]
```diff
@@ -1,50 +1,31 @@
...
-```yaml
-repos:
-  - repo: https://github.com/sourcery-ai/sourcery
-    rev: v1.15.0
-    hooks:
-      - id: sourcery
-        # The best way to use Sourcery in a pre-commit hook:
-        # * review only changed lines:
-        # * omit the summary
-        args: [--diff=git diff HEAD, --no-summary]
```
Comment (Hellebore) [inline README.md:40]: is this information somewhere else (docs?) if people need to find it?

**Review by Hellebore — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,50 +1,31 @@
...
+Sourcery will review any pull request on any GitHub repository to provide instant feedback on the proposed changes.
 
-Get started with our CLI to have Sourcery review all your code locally or star this repo to have our GitHub bot review your most popular Python repo.
+Every review will include a summary of the changes, high level feedback, and line by line suggestions/comments (where relevant).
 
-## CLI
+Our goal is to provide you with the type of code review you would expect to get from a colleague. We're not there yet, but we're continually working to get our reviews closer to that point.
 
-You can have Sourcery review your entire project and make dozens or hundreds (or thousands) of improvements with a single command from the command line. 
+Add Sourcery to any public repo to get reviews for free or you can try it on any private repo for 14 days for free.
```
Comment (brendanator) [inline README.md:10]: **🎯 suggestion:** ‏Remove this sentence

It's duplicated in the next section

File: README.md [hunk 5]
```diff
@@ -1,50 +1,31 @@
 [![Sourcery](https://img.shields.io/badge/Sourcery-enabled-brightgreen)](https://sourcery.ai)
 
-## Sourcery
-Instantly review and improve your Python code to keep your codebase maintainable. 
+## Sourcery Code Reviews
```
Comment (brendanator) [inline README.md:3]: **🎯 suggestion:** Grab the reader's attention‏

```suggestion
## Sourcery - 1000x faster code reviews
```

File: README.md [hunk 6]
```diff
@@ -1,50 +1,31 @@
...
+Sourcery will review any pull request on any GitHub repository to provide instant feedback on the proposed changes.
```
Comment (brendanator) [inline README.md:4]: **🔨 issue:** This section doesn't explain why I should install it

Right now it is an explanation of what Sourcery is. Lead with the value propositions:
- Spend less time reviewing code
- Ensure high quality code
- Automated approvals
- Accelerate your velocity

File: README.md [hunk 7]
```diff
@@ -55,25 +36,18 @@ Find quick installation instructions here:
...
-
-You can get set up and running in 1 minute. Just follow this [link](https://github.com/apps/sourcery-ai/installations/new) and select your repos.
+## Feedback
 
-Or to try it out right away, ⭐ this repo, and Sourcery will review your most popular Python repo and send you a pull request. As an added bonus **@sourcery-ai-bot** will ⭐ your repo too.
-
-## Demo
-
-![Sourcery demo](sourcery-demo.gif)
+We're always trying to learn how we can make Sourcery better. Let us know any thoughts you have by emailing tim@sourcery.ai
```
Comment (brendanator) [inline README.md:41]: **🎯 suggestion:** ‏Use hello@sourcery.ai here so we all get it

**Review by brendanator — CHANGES_REQUESTED** [review]
Right now it's not selling it to me

I'd suggest the following structure
- Attention grabbing title
    - One line super concise explanation of what it is "Instant AI code reviews on all your pull requests"
    - Expound on the benefits
- Getting started
- What is Sourcery?
    - More of an explanation here
    - Screenshots
- ... rest

File: README.md (hunk 2 above)
Comment (angrycaptain19) [reply README.md:72]: They're not relevant to the current functionality. Will re-add when we decide if we're using the existing Sourcery docs or if we are looking to have the review docs live elsewhere

File: README.md [hunk 8]
```diff
@@ -1,50 +1,43 @@
 [![Sourcery](https://img.shields.io/badge/Sourcery-enabled-brightgreen)](https://sourcery.ai)
 
-## Sourcery
-Instantly review and improve your Python code to keep your codebase maintainable. 
+## Sourcery - Instant, AI-Powered, Code Reviews
```
Comment (brendanator) [inline README.md:3]: I'd drop the commas
```suggestion
## Sourcery - Instant AI Code Reviews 
```

**Review by brendanator — APPROVED** [review]

**Pull request merged**